app-frontend/src/app/pages/user/settings/profile/profile.html

```diff
@@ -12,73 +27,157 @@
             <div class="panel-body">
               Your profile picture is managed by
               <span>{{$ctrl.provider.provider === 'gravatar' || $ctrl.provider.provider === 'auth0' ? 'Gravatar' : $ctrl.provider.name}}</span>.
-              <a ng-href="{{$ctrl.provider.link}}" target="_blank">Click here for instructions on how to change it</a>.
+              <a ng-href="{{$ctrl.provider.profilePictureLink}}" target="_blank">Click here for instructions on how to change it</a>.
             </div>
           </div>
         </div>
       </div>
-
-      <div ng-if="$ctrl.provider.provider === 'google-oauth2'">
-        <div class="form-group">
-          <label for="name">Name</label>
-          <input id="name"
+      <div class="form-group">
+        <label for="name">Display name</label>
+        <input id="name"
+               type="text"
+               class="form-control"
+               placeholder="Name"
+               ng-attr-value="{{$ctrl.authService.user.name}}"
+               disabled>
+      </div>
+      <div class="form-group">
+        <label for="username">Username</label>
```

Comment:
The user will not necessarily have an email attached to their account (twitter, etc), and it isn't their username. 
I think it would make more sense to have a "Login" field, which lists their email if it's a direct login via auth0, or their social username and account type for social logins